ff/serving/check_coverage.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Run this script to check code coverage and open results in the browser
+
+# Run Coverage Test
+go test -coverpkg=./... -coverprofile cover.out ./...
+
+# Convert Coverage To HTML
+go tool cover -html=cover.out -o coverage.html
+
+# Cleanup Files
+rm cover.out
+
+# Open In Browser
+open coverage.html
```

Comment:
This may be mac specific

Reply:
Should I submit this first so saadhvi can use it then add support for windows and linux? Or should that all go in one go?

Reply:
Changed To Be Mac Specific